tests/ltp/ltp-batch1/ltp_disabled_tests.txt

```diff
@@ -49,7 +49,7 @@
 #/ltp/testcases/kernel/syscalls/chown/chown05
```

Comment:
These four tests are already disabled and merged in PR https://github.com/lsds/sgx-lkl/pull/812